PureLayout/PureLayout/ALView+PureLayout.m

```diff
@@ -289,6 +310,23 @@ - (NSLayoutConstraint *)autoPinEdgeToSuperviewMargin:(ALEdge)edge relation:(NSLa
     return constraints;
```

Comment:
Couldn't this method be now simplified by using the new `autoPinEdgesToSuperviewMarginsWithInsets` method and passing the `ALEdgeInsets` as zero? This way we could avoid duplicated code.

The same applies to the `autoPinEdgeToSuperviewMargin: withInset:` method.

Reply:
Agreed, `-[autoPinEdgesToSuperviewMargins]` makes sense to simplify with `AlEdgeInsetsZero` but that one differs from the `-[autoPinEdgeToSuperviewMargin: withInset:]` because here the relation does matter and there is no single method currently that does `-[autoPinEdgeToSuperviewMargin: withInset: relation:]`.  Now if something like that is desired then yes, it could simplify things but that would affect backwards compatibility if those two methods were combined into a single one and the singular ones deprecated / removed.  So for now, I can amend the changes for `-[autoPinEdgesToSuperviewMargins]`.  Thanks for the feedback, well appreciated 👍

Reply:
Cool, makes sense 👌.